docs/pages/blog/mui-x-v7-beta.md

```diff
@@ -0,0 +1,245 @@
+---
+title: MUI X v7 is now in beta
+description: Check out what's new and what's next for v7 stable.
+date: 2024-01-29T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+card: true
+---
+
+<div style="max-width: 692px; width: 100%; height: 230px; overflow: hidden; margin-bottom: 16px;">
+  <a href="https://github.com/mui/mui-x/releases/tag/v7.0.0-beta.0">
+    <img src="/static/blog/mui-x-v7-beta/intro.png" alt="MUI X v7 beta release" width="1200" height="400" style="width: 100%; height: 100%; object-fit: cover; object-position: center;" />
+  </a>
+</div>
+
+It's the end of January, and we're ready to start rolling out some of the plans for 2024!
+We have new components, exciting features, and a lot of improvements for both developers and end-users.
+
+Starting [now](https://github.com/mui/mui-x/releases/tag/v7.0.0-beta.0), MUI X v7 is in beta.
+Most breaking changes are in place, and we shift our focus towards refining and expanding over the new version.
+
+## Table of contents
+
+- [Data Grid](#data-grid)
+  - [Sticky headers](#sticky-headers)
+  - [Server-side data source](#server-side-data-source)[<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+  - [Date Object Support in Filter Model](#date-object-support-in-filter-model)
+  - [New stable features](#new-stable-features)
```

Comment:
Optional: Although not substantial, but [this](https://github.com/mui/mui-x/pull/11770) visual update to the columns panel could be a small possible highlight.
```suggestion
  - [New stable features](#new-stable-features)
  - [New columns panel design](#new-columns-panel)
```